src/scene/graph-node.js

```diff
@@ -35,6 +35,8 @@ Object.assign(pc, function () {
         this._dirtyLocal = false;
         this._aabbVer = 0;
 
+        this._frozen = false;
```

Comment:
Would be good to have a definition of what frozen is doing. Expecially in contrast to dirtyWorld/dirtyLocal.

Reply:
By definition, I mean a comment.

Reply:
Yes, I will add the description.